recipe/meta.yaml

```diff
@@ -1,37 +1,41 @@
-{% set version = "0.16.10" %}
-
+{% set version = "0.23.8" %}
 
 package:
   name: awscrt
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/a/awscrt/awscrt-{{ version }}.tar.gz
-  sha256: 3101499b479b82f0403f615bf520c1f8b9506818d275e3e28ade25cc6bcdb92b
+  url: https://pypi.org/packages/source/a/awscrt/awscrt-{{ version }}.tar.gz
+  sha256: cba55f3ee80ea3192a0a24e84caad778570250800a59d29ef9efbcd4d1612f2f
   patches:
-    - 0001-Don-t-force-static-linkage.patch
+    - patches/0001-Don-t-force-static-linkage.patch
 
 build:
-  number: 2
+  number: 0
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - python
     - {{ compiler('c') }}
+    - patch     # [not win]
+    - m2-patch  # [win]
   host:
     - python
     - pip
-    - s2n  # [linux]
-    - aws-checksums
-    - aws-c-common
-    - aws-c-cal
-    - aws-c-io
-    - aws-c-event-stream
-    - aws-c-http
-    - aws-c-auth
-    - aws-c-mqtt
-    - aws-c-s3
+    - setuptools
+    - wheel
+    - s2n 1.5.14  # [linux]
```

Comment:
`aws-lc` is required on `linux` too https://github.com/awslabs/aws-crt-python/blob/v0.23.8/setup.py#L169,  see also https://github.com/awslabs/aws-crt-python/blob/a65aadb901f0658eeec6b59fbb5f56e54d84282e/README.md?plain=1#L47 and https://github.com/aws/aws-lc

Reply:
Thank you, Serhii!

Reply:
AWS-LC is a fork of BoringSSL tailored for use within AWS products. It supplies the cryptographic primitives (e.g. libcrypto.a) needed by components like s2n.
Here we provide our own s2n, which relies on openssl for its cryptographic primitives. So we do not need and do not want to include aws-lc.